tests/scripts/ci/test_link_rustume_mentions.py

```
@@ -0,0 +1,32 @@
+"""Tests for apps/site/scripts/link-rustume-mentions.py."""
+
+from __future__ import annotations
+
+import importlib.util
+from pathlib import Path
+
+_MODULE_PATH = (
+    Path(__file__).resolve().parents[3] / "apps" / "site" / "scripts" / "link-rustume-mentions.py"
+)
+_spec = importlib.util.spec_from_file_location("link_rustume_mentions", _MODULE_PATH)
+assert _spec and _spec.loader
```

Ruff (0.15.14)

[warning] Assertion should be broken down into multiple parts

Break down assertion into multiple parts

(PT018)